api/v1alpha3/types.go

```diff
@@ -693,6 +693,10 @@ type Volume struct {
 	// +optional
 	IOPS int64 `json:"iops,omitempty"`
 
+	// Throughput to provision in MB/s supported for the volume type. Not applicable to all types.
+	// +optional
+	Throughput int64 `json:"throughput,omitempty"`
```

Comment:
As this change will be going into main it will be for v1alpha4 only (i.e. 0.7.0 release). Therefore, would you be able to remove the changes to the v1alpha3 api files?

If needed we can always backport to the 0.6.x releases (i.e. v1alpha3) by creating a separate PR later and cherry pick the change

(we can backport this to v1alpha3 which

Reply:
Sure, I previously did not include the v1alpha3 struct changes, I can revert that. It will make the conversion tooling complain, I'm not sure what the correct action is there.

Reply:
As this is a new field in v1alpha4 you will have to modify the conversion functions. I had to do something similar for adding taints:

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/exp/api/v1alpha3/conversion.go#L57
https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/exp/api/v1alpha3/conversion.go#L167

Reply:
Feel free to ping me if you need a hand

Reply:
I don't have a good understanding of how to satisfy the conversion tooling. I added a function to reference the autoconvert func, but the tooling is unsatisfied. Any more involved fixes I attempt is erased by `make generate`.

Reply:
@cnmcavoy - i can have a look. What would be best with you out of the 2:

1) Add me as a contributor to your branch and i can add a commit
or
2) Have a zoom call to pair

Reply:
I added you as a contributor on the repo

Reply:
Thanks - having a look now

Reply:
Theres an issue with conversion-gen thats causing these problems. I am going to make some changes to out API types in #2494 and that should help then.

Reply:
thanks for looking! I'll take another stab at the conversion after #2495 is merged.

Reply:
I rebased + added the conversion funcs. It looks like the only tests that fail now are the fuzzing tests. Not sure what to do there to make the fuzzing work.

Reply:
Just pushed a change to your branch. Hopefully the fuzzing tests will pass now :crossed_fingers: